db/crud.go

```diff
@@ -1682,6 +1704,69 @@ func (db *DatabaseCollectionWithUser) revTreeConflictCheck(ctx context.Context,
 	return parent, currentRevIndex, nil
 }
 
+func (db *DatabaseCollectionWithUser) ResolveHLVConflict(ctx context.Context, localDoc, incomingDoc *Document, resolver *ConflictResolver) (*HybridLogicalVector, error) {
```

Comment:
What is the benefit to having this function vs `DatabaseCollectionWithUser.resolveConflict` function? If I understand correctly this function is basically a duplicate except that it will update the HLV.